critiques/deferred-observations.md

```
@@ -1519,7 +1519,17 @@ against all five mixins (`critiques/phase5-validation.md`, PR-19 §11).
 
 ### Added by the PR-22 adversarial review (round 2)
 
-65. **The "modules < 1000 lines" waiver names `pdsfile.py` and the rule modules,
+65. **RESOLVED — owner decision, 2026-08-03**
```

markdownlint-cli2 (0.23.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 65; Style: 1/1/1

(MD029, ol-prefix)